Sources/Filters/General/OBBTree/test/testOBBTree.js

```diff
@@ -0,0 +1,192 @@
+import test from 'tape-catch';
+
+import vtkArrowSource from 'vtk.js/Sources/Filters/Sources/ArrowSource';
+import vtkCubeSource from 'vtk.js/Sources/Filters/Sources/CubeSource';
+import vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import vtkOBBTree from 'vtk.js/Sources/Filters/General/OBBTree';
+import vtkMatrixBuilder from '../../../../Common/Core/MatrixBuilder';
+import vtkPolyData from '../../../../Common/DataModel/PolyData';
+import vtkTriangleFilter from '../../TriangleFilter';
```

Comment:
Rewrite to use absolute paths from vtk.js root, so it's not confusing as to where the symbol comes from.